### PR TITLE
Feature/import polygon

### PR DESCRIPTION
**Pull request by lukahartwig**

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -15,18 +13,15 @@ module.exports = ({ config }) => {
                         prettierConfig: {
                             printWidth: 150,
                             tabWidth: 5,
```
Comment (LeonardKoch) [inline .storybook/webpack.config.js]: Tabwidth 5 seems rather unusual to me. Is that correct?

File: src/PolygonDraw/reducer.ts [hunk 2]
```diff
@@ -32,6 +33,14 @@ export const polygonEditReducer = (state: PolygonEditState, action: Actions): Po
                 polygons: [...action.payload]
             };
         }
+        case SET_POLYGON: {
+            const newPolygons = state.polygons.slice();
```
Comment (medhoover) [inline src/PolygonDraw/reducer.ts]: I think the polygon with the active index should be replaced

Comment (lukahartwig) [reply src/PolygonDraw/reducer.ts]: That's what's happening in the next line. `.slice()` is just a way of creating a copy of the array.

File: .storybook/webpack.config.js (hunk 1 above)
Comment (lukahartwig) [reply .storybook/webpack.config.js]: Good eye. Fixed.

**Pull request merged**